docs/multiple-accounts.md

```diff
@@ -0,0 +1,239 @@
+# Multiple Accounts with the CLI - v2.40.0
+
+Since its creation, `gh` has enforced a mapping of one account per host. Functionally, this meant that when targeting a
+single host (e.g. github.com) each `auth login` would replace the token being used for API requests, and for git
+operations when `gh` was configured as a git credential manager. Removing this limitation has been a [long requested
+feature](https://github.com/cli/cli/issues/326), with many community members offering workarounds for a variety of use cases.
+A particular shoutout to @gabe565 and his long term community support for https://github.com/gabe565/gh-profile in this space.
+
+With the release of `v2.40.0`, `gh` has begun supporting multiple accounts for some use cases on github.com and
+in GitHub Enterprise. We recognise that there are a number of missing quality of life features, and we've opted
+not to address the use case of automatic account switching based on some context (e.g. `pwd`, `git remote`, etc).
+However, we hope many of those using these custom solutions will now find it easier to obtain and update tokens (via the standard
+OAuth flow rather than as a PAT), and to store them securely in the system keyring managed by `gh`.
+
+We are by no means excluding these things from ever being native to `gh` but we wanted to ship this MVP and get more
+feedback so that we can iterate on it with the community.
```

Comment:
**How does this version read?**  Leaving diving into the nuances in subsequent sections, I hope this can give a high-level summary that people can get the gist.

```suggestion
With the `v2.40.0` release, [multi-account support](https://github.blog/changelog/2023-11-03-multi-account-support-on-github-com/) is introduced to `gh` for GitHub.com and
in GitHub Enterprise and it is only the beginning!

Thanks to @gabe565 for his efforts to support multi-account capabilities with [`gh-profile` extension](https://github.com/gabe565/gh-profile) for this [long requested feature](https://github.com/cli/cli/issues/326)! ❤️ 

This release focuses on several specific use cases:

- adding multiple accounts for GitHub.com and GitHub Enterprise with `gh auth login`
- switching manually between accounts for `gh` and `git` use with `gh auth switch`
- seeing multiple accounts with `gh auth status`
- logging out of accounts with `gh auth logout`

Future enhancements such as automatic account switching based on contexts and more are possible based upon user feedback!
```

_Really want to link "user feedback" to where people can click a link and leave said feedback._

Reply:
So, the tone of your suggested edits feels like release notes to me, and I think that it does a good job at that. However, I don't see this document as release notes but more a deep dive for the reader who wants more precision in what has actually changed.

How would you feel if we took your suggestion and used it in the release notes (and included a link to the release discussion for feedback), linking to this document?